test/androidTest/java/org/thoughtcrime/securesms/util/Expectations.java

```diff
@@ -19,6 +21,10 @@ public static void checkIsDisplayed(int id) {
         onView(withId(id)).check(matches(isDisplayed()));
     }
 
+    public static void checkIsDisplayedWithRearCameraDrawable(int id){
```

Comment:
Could make the naming more generic "checkIsDisplayedWithResource"

Reply:
This is actually a method that i'm no longer using. Will remove